src/main/infrastructure/repository/iab/IABVendorListRepository.js

```diff
@@ -2,15 +2,19 @@ import {VendorListRepository} from '../../../domain/vendorlist/VendorListReposit
 import {VendorList} from '../../../domain/vendorlist/VendorList'
 import {inject} from '../../../core/ioc/ioc'
 import {GVLFactory} from './GVLFactory'
+import {Language} from '../../../domain/vendorlist/Language'
 
 export class IABVendorListRepository extends VendorListRepository {
   constructor({gvlFactory = inject(GVLFactory)} = {}) {
     super()
     this._gvlFactory = gvlFactory
   }
 
-  async getVendorList({version} = {}) {
-    const gvl = this._gvlFactory.create({version: version?.value})
+  async getVendorList({version, language = new Language('en')} = {}) {
```

Comment:
Is this default 'en' language necessary? because the GVL factory has already english as default language, right?

Reply:
I have just changed GVLFactory. 
In some places of the BorosTcf we are accessing to GVLFactory, sometimes through IABVendorListRepository and other times directly. When you access directly (without IABVendorListRepository) you always use english. May be we can do a litle refactor for avoiding accessing directly GVLFactory and always use  IABVendorListRepository.

Reply:
I agree with the refactor